### DISCUSSION TITLE
Motioneye .eo

### DISCUSSION THREAD

**Discussion by Schnitzer50** [community/community#57234] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Sehr geehrter Herr Michault.
ich habe gemäß Ihrer Anleitung MotionEye.eo auf meinem Raspberry Pi 4B mit Debian Bulleye installiert.
Soweit hat alles funktioniert. Die USB Kamera wurde problemlos erkannt.
Jetzt habe ich ein Problem mit der neuen Rasperry Pi Camera, Module 3.
Diese Kamera wird von MotionEye.eo nicht angesprochen. Ich bekomme kein Bild. In den Einstellungen komme ich nicht weiter.
Mit dem Befehl: libcamera-hello -t 0 kann ich ein Bild auf dem Monitor erzeugen.
Welche Einstellungen muß ich bei MotionEye.eo
Mit freundlichen Grüßen
W. Stock

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/jmichault/motioneye.eo? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:

**Comment by Schnitzer50** [comment]
Danke für die Info.
Habe die Frage weitergeleitet.
